crates/iceberg/src/spec/table_properties.rs

```diff
@@ -49,6 +56,10 @@ pub struct TableProperties {
     pub write_format_default: String,
     /// The target file size for files.
     pub write_target_file_size_bytes: usize,
+    /// Compression codec for Avro files (manifests, manifest lists)
+    pub avro_compression_codec: String,
+    /// Compression level for Avro files (None uses codec-specific defaults: gzip=9, zstd=1)
```

Comment:
```suggestion
    /// Compression level for Avro files (None uses codec-specific defaults)
```
nit: these might change in the future, so i think its better to not specify the values here 

there are a few other instances of this same comment